packages/headless/src/ssr-commerce.index.ts

```diff
@@ -0,0 +1,222 @@
+import {polyfillCryptoNode} from './api/analytics/analytics-crypto-polyfill';
+
+polyfillCryptoNode();
+
+// 3rd Party Libraries
+export type {Unsubscribe, Middleware} from '@reduxjs/toolkit';
+export {createAction, createAsyncThunk, createReducer} from '@reduxjs/toolkit';
+export type {AnalyticsClientSendEventHook} from 'coveo.analytics';
+export type {Relay} from '@coveo/relay';
+
+// Main App
+export type {
+  CommerceEngineOptions,
+  CommerceEngineConfiguration,
+} from './app/commerce-engine/commerce-engine';
+export type {
+  SSRCommerceEngine as CommerceEngine,
+  CommerceEngineDefinition,
+  CommerceEngineDefinitionOptions,
+} from './app/commerce-engine/commerce-engine.ssr';
+export {defineCommerceEngine} from './app/commerce-engine/commerce-engine.ssr';
+export {getSampleCommerceEngineConfiguration} from './app/commerce-engine/commerce-engine-configuration';
+
+// export type
+export type {CoreEngine, ExternalEngineOptions} from './app/engine';
+export type {
```

Comment:
We should exporte CommerceEngineConfiguration from .app/commerce-engine/commerce-engine-configuration